src/app/core/services/config/remote-config.service.ts

```diff
@@ -108,13 +115,18 @@ export class FirebaseRemoteConfigService extends RemoteConfigService {
 
   private fetch(timeoutMillis: number) {
     console.log('Fetching Firebase Remote Config')
+    if (!this.platform.is('cordova'))
+      return Promise.resolve(this.configSubject.value)
```

Comment:
Note: this will still fetch a `FirebaseRemoteConfig` object from `configSubject`. Perhaps replace the `configSubject` initial value with an `EmptyRemoteConfig` object? E.g.
```typescript
class EmptyRemoteConfig: RemoteConfig {
  get(key: ConfigKeys) {
    return Promise.resolve()
  }
  getOrDefault(key: ConfigKeys, defaultValue: any) {
    return Promise.resolve(defaultValue)
  }
}
```

Reply:
Made the changes.